solver/src/solver/oneinch_solver.rs

```diff
@@ -163,7 +164,10 @@ impl Interaction for Swap {
 
 #[async_trait::async_trait]
 impl SingleOrderSolving for OneInchSolver {
-    async fn settle_order(&self, order: LimitOrder) -> Result<Option<Settlement>> {
+    async fn try_settle_order(
```

Comment:
does this mean that the 1Inch solver now has retry logic (when it previously didn't)?

Reply:
Yes sir. Although it remains to creat an error class with at least one retryable error.

Reply:
So at this moment there are actually no retryable errors for 1inch so it would never actually retry.